File: mk_1/app/src/dominio/TransformacionFichero.java
package dominio;

import java.io.*;
import java.lang.*;
import java.nio.file.*;

public class TransformacionFichero {

	public void copiarFileStr(String fileName, String str) { 
        try { 
        	Files.write(Paths.get(fileName), str.getBytes(), StandardOpenOption.APPEND);
        }
        catch (IOException e) { 
            System.out.println("Exception occoured" + e); 
        }
    }

    public void copiarFile(String fileName) { 
        try { 
        	Files.copy(Paths.get(fileName), Paths.get("./test_data/test.arff"), 
        		StandardCopyOption.REPLACE_EXISTING);
        }
        catch (IOException e) { 
            System.out.println("Exception occoured" + e); 
        }
    }

    public void borrarFile(String fileName) { 
        try { 
        	Files.deleteIfExists(Paths.get(fileName));
        }
        catch (IOException e) { 
            System.out.println("Exception occoured" + e); 
        }
    }
}

File: mk_2/app/src/dominio/transformacion_fichero.py
import shutil
import os

def copiar_file_str(file_name, string):
    try:
        f = open(file_name, "a")
        f.write(string)
    except:
        print("Error al copiar string en el fichero")

def copiar_file(file_name):
    try:
        shutil.copyfile(file_name, "../test_data/test.arff")
    except:
        print("Error al copiar el fichero en './test_data/test.arff'")

def borrar_file(file_name):
    try:
        os.remove(file_name)
    except:
        print("Error al eliminar fichero")
